src/journal/journalctl.c

```diff
@@ -380,6 +382,7 @@ static int help(void) {
                "  -u --unit=UNIT             Show logs from the specified unit\n"
                "     --user-unit=UNIT        Show logs from the specified user unit\n"
                "  -t --identifier=STRING     Show entries with the specified syslog identifier\n"
+               "  -T --exclude-identifier=STRING Hide entries with the specified identifier\n"
```

Comment:
Please break the line if the string does not fit the current alignment.

Reply:
Thanks! What is the convenient way to break the line? I didn't find any long options as a reference not fitting the line, so I'm not sure about break position and indentation.

Reply:
You can check `systemctl`'s help output and look for the `--check-inhibitors=` option`

Reply:
Thanks a lot, done!